Common.mk

```diff
@@ -284,6 +282,10 @@ define BINARY_TARGET_BODY
 
 endef
 
+define GET_CLONE_URL
+	$(shell source $(BUILD_LIB)/common.sh && build::common::get_clone_url $(1) $(2) $(AWS_REGION) $(CODEBUILD_CI))
```

Comment:
So i just ran into this too, this space is an issue, which is why you have the strip above.  Lets just remove these leading space/tab, comment as to why if you want to.